### ISSUE TITLE
FirstUsePage spacing between buttons is to wide

### ISSUE THREAD

**Issue by Rebecca367** [openmsupply/mobile#2547]
## Describe the bug
Initial page is broken after closing wrong alert message on initial page. 
<img width="565" alt="Screen Shot 2020-03-16 at 1 09 38 PM" src="https://user-images.githubusercontent.com/57238075/76713868-8c94da00-6788-11ea-9096-c6157388c419.png">

### To reproduce

1. Go to 'request a demo store'
2. Click on 'close' back to initialisation page 
3. See error

### Expected behaviour

initialisation page should not be broken 

### Version and device info

- App version: v4.0.2
- Tablet model: emulator
- OS version:

**Comment by joshxg** [comment]
@Rebecca367 can you provide more details please? Just saying "it's broken" is insufficient

**Comment by Rebecca367** [comment]
@joshxg 
Image above explains that it's broken which means not normal image, so I don't know what else do you want me to provide?

**Comment by joshxg** [comment]
@Rebecca367 what does "broken" mean?

**Comment by Rebecca367** [comment]
<img width="468" alt="Screen Shot 2020-03-16 at 1 33 27 PM" src="https://user-images.githubusercontent.com/57238075/76714226-d67ebf80-678a-11ea-914e-34209ae04fc7.png">
@joshxg 
If you are not happy with the word "broken" then I can rephrase it to "extra space between connect and demo a store button". It is a new bug that has shown on my side, if you  couldn't replicate this issue. I can help with sending datafile that I am using to you.

**Comment by joshxg** [comment]
Ah, so the space between the buttons is the problem - now I understand

**Comment by Rebecca367** [comment]
The spacing area between buttons has reduced for some reasons and FirstUserPage looks better now. So, I will close this issue now.
<img width="546" alt="Screen Shot 2020-03-25 at 10 03 17 PM" src="https://user-images.githubusercontent.com/57238075/77519642-b4044900-6ee4-11ea-8291-1059e3456776.png">
